src/constants/featureFlags.ts

```diff
@@ -1,3 +1,4 @@
 export const FEATURE_FLAGS = {
   GGS_WHITELISTED_REPOS: "ggs_whitelisted_repos",
```

Comment:
Should we remove this since this is no longer in use?

Reply:
we cannot remove until this feature is released cos prod still uses this flag

Reply:
let's add a todo above

Reply:
removed in code, added deploy note to remove in gb